src/lv_api_map.h

```diff
@@ -137,7 +136,7 @@ static inline void lv_obj_move_background(lv_obj_t * obj)
 #define lv_disp_get_layer_top            lv_display_get_layer_top
 #define lv_disp_get_layer_sys            lv_display_get_layer_sys
 #define lv_disp_get_layer_bottom         lv_display_get_layer_bottom
-#define lv_disp_add_event                lv_display_add_event
+#define lv_disp_add_event                lv_display_add_event_cb
```

Comment:
Display events were introduced in v9, so no v8 mapping is required. Please remove `lv_disp_add_event`, `lv_disp_get_event_count`, `lv_disp_get_event_dsc`, and `lv_disp_remove_event`